modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/message/SnapshotMetaResponse.java

```diff
@@ -22,9 +22,7 @@
 import org.apache.ignite.internal.table.distributed.TableMessageGroup;
 import org.apache.ignite.raft.jraft.entity.RaftOutter.SnapshotMeta;
 
-/**
- * Snapshot meta response message.
- */
+/** Snapshot meta response message. */
```

Comment:
Why did you do this? Please explain, this comment was absolutely fine, and you didn't change anything else in the file.